crypto/fipsmodule/aes/mode_wrappers.c

```diff
@@ -138,6 +138,14 @@ void AES_cfb128_encrypt(const uint8_t *in, uint8_t *out, size_t length,
   *num = (int)num_u;
 }
 
+#if defined(GHASH_ASM_X86_64)
+int crypto_xts_avx512_enabled(void) {
+  return (CRYPTO_is_VAES_capable() &&
+          CRYPTO_is_AVX512_capable() &&
+          CRYPTO_is_VPCLMULQDQ_capable());
```

Comment:
don't you need `vbmi2` as well for this implementation?

Reply:
I don't think we need that, we are not using instruction from vbmi2, https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#ig_expand=6,1915,7675,1743&avx512techs=AVX512_VBMI2

Reply:
looks like we are using one instruction from vbmi2 family, will update the check. Thank you for pointing it out.